frr-10.1.yaml

```
@@ -0,0 +1,110 @@
+package:
+  name: frr-10.1
+  version: 10.1.1
+  epoch: 0
+  description: The FRRouting Protocol Suite
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    provides:
+      - frr=${{package.full-version}}
+    runtime:
+      - python-3.11
```

```suggestion
      - bash
      - iproute2
      - python3
```

Looks like these are all needed: https://github.com/FRRouting/frr/blob/master/alpine/APKBUILD.in#L9 - the python3 thing we should be able to get away with just `3` - I don't see anything 3.11 specific in their code.